packages/flutter/lib/src/painting/image_provider.dart

```diff
@@ -1007,7 +1007,7 @@ class MemoryImage extends ImageProvider<MemoryImage> {
 /// Then, to fetch the image and associate it with scale `1.5`, use:
 ///
 /// ```dart
-/// AssetImage('icons/heart.png', scale: 1.5)
+/// ExactAssetImage('icons/heart.png', scale: 1.5)
```

Comment:
This should probably be wrapped in a `{@tool snippet}` directive so the analyzer would have caught that AssetImage is wrong here. Can you try that?

Reply:
Of course!